docs/update.md

```diff
@@ -263,7 +263,49 @@ You can also delete the `PerconaPGUpgrade` resource (this will clean up the jobs
 $ kubectl delete perconapgupgrade cluster1-15-to-16
 ```
 
-If there are [custom PostgreSQL extensions](custom-extensions.md) installed in the cluster, they need to be taken into account: you need to build and package each custom extension for the new PostgreSQL major version. During the  upgrade the Operator will install extensions into the upgrade container.
+#### Upgrading PostgreSQL extensions
+
+If there are [custom PostgreSQL extensions](custom-extensions.md#adding-custom-extensions) installed in the cluster, they need to be taken into account: you need to build and package each custom extension for the new PostgreSQL major version. During the  upgrade the Operator will install extensions into the upgrade container.
+
+The only [built-in extension](custom-extensions.md#enabling-or-disabling-built-in-extensions) which demands special treatment after the database upgrade is `pg_stat_monitor` one. It is used to provide query analytics for Percona Monitoring and Management (PMM), if enabled in the Custom Resource (`deploy/cr.yaml` manifest):
+
+```yaml
+extensions:
+  ...
+  builtin:
+    pg_stat_monitor: true
```

Comment:
@fiowro We need to mention that we will automate it starting from PGO 2.6.0

Reply:
done